Gems/Atom/RHI/Code/Include/Atom/RHI/ShaderResourceGroupData.h

```diff
@@ -212,6 +212,7 @@ namespace AZ
                 ReadWriteTexture,
                 ReadBuffer,
                 ReadWriteBuffer,
+                ReadTextureCube,
```

Comment:
I only see updates to Vulkan. What about other backends?

Reply:
No changes necessary for DX12.